### DISCUSSION TITLE
Scalable Ops

### DISCUSSION THREAD

**Discussion by wirmachenbunt** [cables-gl/cables_docs#779] [Feature Requests] [answered]
the sequence op can be scaled now, which is awesome. but there are other ops like transform which need this as well. patching simple xyz stuff makes patches hard to read since all cables need to be fed into the cramped space above transform

![image](https://github.com/cables-gl/cables_docs/assets/1138673/86d651d8-ecf7-43b8-a6c2-f2615a0c4372)

**Comment by wirmachenbunt** [comment]
as a comparison where this is coming from
![image](https://github.com/cables-gl/cables_docs/assets/1138673/0bcaa402-6d7a-4d09-bcfa-2dd8533b3444)

**Comment by pandrr** [comment, marked as answer]
on dev.cables.gl you can select an op and press ctrl/cmd+p and search for "resizable" 

not 100% happy with it. how is it in vvvv ? are inputs and outputs stretched out ? or can you control this ?
